### PR TITLE
fix: RTL chevron rotation

### PR DESCRIPTION
**Pull request by Goudarz**

### CODE REVIEW

File: layouts/partials/breadcrumb.html [hunk 1]
```diff
@@ -4,7 +4,11 @@
       <div class="whitespace-nowrap transition-colors min-w-[24px] overflow-hidden text-ellipsis hover:text-gray-900 dark:hover:text-gray-100">
         <a href="{{ .Permalink }}">{{ .LinkTitle }}</a>
       </div>
-      {{- partial "utils/icon.html" (dict "name" "chevron-right" "attributes" "class=\"w-3.5 shrink-0\"") -}}
+      {{ if eq .Site.Language.LanguageDirection "rtl" }}
+        {{- partial "utils/icon.html" (dict "name" "chevron-left" "attributes" "class=\"w-3.5 shrink-0\"") -}}
+      {{ else }}
+        {{- partial "utils/icon.html" (dict "name" "chevron-right" "attributes" "class=\"w-3.5 shrink-0\"") -}}
+      {{ end }}
```
Comment (imfing) [inline layouts/partials/breadcrumb.html:11]:
```suggestion
      {{- partial "utils/icon.html" (dict "name" "chevron-right" "attributes" "class=\"w-3.5 shrink-0 rtl:-rotate-180\"") -}}
```

File: layouts/partials/components/pager.html [hunk 2]
```diff
@@ -44,7 +44,11 @@
         class="flex max-w-[50%] items-center gap-1 py-4 text-base font-medium text-gray-600 transition-colors [word-break:break-word] hover:text-primary-600 dark:text-gray-300 md:text-lg ltr:ml-auto ltr:pl-4 ltr:text-right rtl:mr-auto rtl:pr-4 rtl:text-left"
       >
         {{- $next.LinkTitle -}}
-        {{- partial "utils/icon.html" (dict "name" "chevron-right" "attributes" "class=\"inline h-5 shrink-0\"") -}}
+        {{ if eq .Site.Language.LanguageDirection "rtl" }}
+          {{- partial "utils/icon.html" (dict "name" "chevron-left" "attributes" "class=\"inline h-5 shrink-0\"") -}}
+        {{ else }}
+          {{- partial "utils/icon.html" (dict "name" "chevron-right" "attributes" "class=\"inline h-5 shrink-0\"") -}}
+        {{ end }}
```
Comment (imfing) [inline layouts/partials/components/pager.html:51]:
```suggestion
        {{- partial "utils/icon.html" (dict "name" "chevron-right" "attributes" "class=\"inline h-5 shrink-0 rtl:-rotate-180\"") -}}
```

**Review by imfing — CHANGES_REQUESTED** [review]
in this case, we could just use Tailwind CSS to rotate the chevron icon:

[Handling Hover, Focus, and Other States - Tailwind CSS](https://tailwindcss.com/docs/hover-focus-and-other-states#rtl-support)

you may need to run `npm run build:css` to compile the CSS styles and commit it after the above changes have been made.

**Comment by Goudarz** [comment]
@imfing, The files has been updated according to the documentation. 👍

**Review by imfing — APPROVED** [review]
@Goudarz awesome, thank you!

**Pull request merged**